README.md

```diff
@@ -46,6 +44,33 @@
 
 ![Deploy](resources/Deploy.gif)
 
+## Language-Specific Prerequisites
+
+### JavaScript
+
+* [Node 8.0+](https://nodejs.org/)
+
+### C#
+
+* [VS Code Debugger for C#](https://marketplace.visualstudio.com/items?itemName=ms-vscode.csharp)
+* [Dotnet cli](https://docs.microsoft.com/dotnet/core/tools/?tabs=netcore2x)
```

Comment:
nit: Typically CLI is capitalized and all documentation refers to it as .NET instead of spelling it phonetically.  Though I see that the command for it is "dotnet"